contracts/exchangeIssuance/FlashMintHyETHV3.sol

```diff
@@ -378,7 +375,8 @@ contract FlashMintHyETHV3 is Ownable, ReentrancyGuard {
             // Special handling for agETH
             if (marketData.underlying == address(agETH)) {
                 rsEthAdapter.getRSETHWithETH{value: ethAmount}("");
-                marketData.sy.deposit(address(this), address(agETH), ethAmount, 0);
+                uint256 agEthAmount = agETH.balanceOf(address(this));
```

Comment:
The amount of agETH returned from the rsETH adapter is different than `ethAmount`, so I get the balance and use that instead.